handouts/Session_14.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Phylogenies in Computational Historical Linguistics\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "### 1.1 Inner and Outer Language History\n",
+    "The origins of phylogenetics go back to the 19th century and its principles haven't changed much since. Among the lesser known early linguists there is Georg von der Gabelentz (1840-1893). The following description of phylogenetic reconstruction which is based on his words from 1891 may still hold true:\n",
+    "\n",
+    ">This branch of linguistics is at first concerned with the most dry individual facts: Are languages A and related and to what degree? Does this word or word form exist in that language or at that time of language history? How is it pronounced there? What kinds of rules can be found in the sound differences? Is it a native or a loan word? Which words belong to the original lexicon, which have been added later? And so on. All this sounds and really is very dry. The innermost drive of human language, that which otherwise makes comparative linguistics one of the most lively subjects, stands back at first: Merely a few of the field's offshoots wind to the mental and social life of the speakers. A linguist focused on a single language hurries to absorb it into themself whereas a language historian stands outside of it, like an anatomist next to a corpse.\n",
+    "[...]\n",
+    "We best should distinguish between outer and inner language history. Outer language history is the history of its spacial and temporal spreading, of its branchings and possible interminglings - in other words, its genealogy. Inner language history shows and tries to explain how the language slowly changed in regards to meaning and form.\"\n",
+    "(Gabelentz 1891: 145f., translated/rephrased)\n",
+    "\n",
+    "### 1.2 Trees, Waves, and Nets\n",
+    "   \n",
+    "#### Trees\n",
+    "\n",
+    "This first phylogenetic tree on the Slawic languages by František Čelakovský (1799-1852) was published in 1853, shortly before August Schleier published his tree model theory in the same year:\n",
+    "\n",
+    "![alt text](https://github.com/digling/calc-seminar/blob/master/data/S14.1.png)\n",
+    "\n",
+    ">The oldest splits of Indo-European until the creation of the basic languages of which the language trunk consists can be illustrates by this diagram. The length of lines imply time depth, their distance the degree of relationship.\n",
+    "(Schleicher 1861: 6, translated/rephrased)\n",
```

Comment:
As a general rule, if you translate something (rephrase is not really allowed if you put quotation marks around it), you should write "my translation", thus: Schleicher 1861: my translation, or in case of multi-authors: "our translation", or "translation by NES".